src/main/scala/org/apache/spark/deploy/armada/submit/ArmadaClientApplication.scala

```diff
@@ -664,19 +640,21 @@ private[spark] class ArmadaClientApplication extends SparkApplication {
   ): Seq[String] = {
     val executorsResponse = armadaClient.submitJobs(queue, jobSetId, executors)
     executorsResponse.jobResponseItems.map { item =>
-      log(s"Submitted executor job with ID: ${item.jobId}, Error: ${item.error.orElse("none")}")
+      val error = if (item.error.nonEmpty) item.error else "none"
```

Comment:
```suggestion
      val error = item.error.nonEmpty.getOrElse("none")
```

Reply:
Agree with the suggestion - a nice simplification.